### PR TITLE
Add hiking osm on project page

### PR DESCRIPTION
**Pull request by nobohan**
Hello, 

This a PR for announcing the hiking-osm-be project! I'm planning to make some announcements and to "publicly" release the hiking.osm.be page beginning of next week. 

Julien

### CODE REVIEW

File: en/projects/hiking-osm-be.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: hiking-osm-be maps
```
Comment (jbelien) [inline en/projects/hiking-osm-be.md:2]: Needs a title a bit more "sexy" 😁

File: nl/projects/hiking-osm-be.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: hiking-osm-be maps
```
Comment (jbelien) [inline nl/projects/hiking-osm-be.md:2]: Needs a title a bit more "sexy" 😁

File: en/projects/hiking-osm-be.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+These maps, whose catalog will evolve with time and demand, can be downloaded at a free price from [hiking.osm.be](https://hiking.osm.be/). Benefiting from the continuous updating of OpenStreetMap data, these maps are regularly updated. In particular, all [marked routes](https://wiki.openstreetmap.org/wiki/WikiProject_Belgium/Conventions/Walking_Routes) displayed on the map are derived from OpenStreetMap data.
+
+By contributing to OpenStreetMap, you improve these maps and can use them freely for your leisure!
+
+This project was initiated by the two companies [Champs-Libres Coopérative](https://www.champs-libres.coop/) and the [atelier cartographique](https://atelier-cartographique.be/). More information on the [FAQ](https://hiking.osm.be/fr/index.html#FAQ).
```
Comment (jbelien) [inline en/projects/hiking-osm-be.md:16]:
```suggestion
This project was initiated by the two companies [Champs-Libres Coopérative](https://www.champs-libres.coop/) and the [atelier cartographique](https://atelier-cartographique.be/). More information on the [FAQ](https://hiking.osm.be/en/index.html#FAQ).
```

File: nl/projects/hiking-osm-be.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+These maps, whose catalog will evolve with time and demand, can be downloaded at a free price from [hiking.osm.be](https://hiking.osm.be/). Benefiting from the continuous updating of OpenStreetMap data, these maps are regularly updated. In particular, all [marked routes](https://wiki.openstreetmap.org/wiki/WikiProject_Belgium/Conventions/Walking_Routes) displayed on the map are derived from OpenStreetMap data.
+
+By contributing to OpenStreetMap, you improve these maps and can use them freely for your leisure!
+
+This project was initiated by the two companies [Champs-Libres Coopérative](https://www.champs-libres.coop/) and the [atelier cartographique](https://atelier-cartographique.be/). More information on the [FAQ](https://hiking.osm.be/fr/index.html#FAQ).
```
Comment (jbelien) [inline nl/projects/hiking-osm-be.md:16]:
```suggestion
This project was initiated by the two companies [Champs-Libres Coopérative](https://www.champs-libres.coop/) and the [atelier cartographique](https://atelier-cartographique.be/). More information on the [FAQ](https://hiking.osm.be/nl/index.html#FAQ).
```

**Review by jbelien — CHANGES_REQUESTED** [review]

**Comment by jbelien** [comment]
Don't forget to translate <https://hiking.osm.be/>.

**Review by jbelien — APPROVED** [review]
Good to go !
Let me know when you want me to merge it 😃

**Comment by nobohan** [comment]
Hello Jonathan, it is a good time to merge it. I'll make some announcements today in one hour.

**Pull request merged**